istio/README.md

```diff
@@ -7,13 +7,14 @@ You will be using add-ons like Jaeger, Prometheus, Grafana, Servicegraph & Weave
 ## Pre-req
 
 1. **Your cluster needs at least 4 CPUs to install Istio and the JPetStore application.** If you are using an existing cluster with less than 4 CPUs, add worker nodes to increase the cluster capacity.
-1. Follow the instructions in the parent [README](../README.md) to deploy the secrets and applications using helm.
-1. *Uninstall* the JPetStore applications with helm (you will reinstall them later once Istio is enabled):
+2. Follow the instructions in the parent [README](../README.md) to deploy the secrets and applications using helm.
+3. *Uninstall* the JPetStore applications with helm (you will reinstall them later once Istio is enabled):
 
    ```sh
    helm delete jpetstore --purge
    helm delete mmssearch --purge
    ```
+4. Upgrade Helm to v2.10. Use `helm version` to check the client and server's versions.
```

Comment:
Maybe we can add a link to [install latest client version](https://docs.helm.sh/using_helm/#installing-helm) and `helm init --upgrade` to upgrade tiller component